__tests__/app.test.js

```diff
@@ -146,6 +147,35 @@ describe('/api/articles/:article_id', () => {
     })
 })
 
+describe('/api/articles', () => {
+    test('GET 200: responds with an array of objects of articles, each with the correct properties', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body: {articles}}) => {
+            articles.forEach((article) => {
```

Comment:
As we are using a forEach here - if the array is empty then this test will pass by default as none of the expect statements will be called. A test with no expect statements will pass by default. We should first check that there is something in the array (check its length)